tests/cupy_tests/statics_tests/test_histogram.py

```diff
@@ -148,3 +148,138 @@ def test_bincount_zero(self, xp, dtype):
     def test_bincount_too_small_minlength(self, xp, dtype):
         x = testing.shaped_arange((3,), xp, dtype)
         return xp.bincount(x, minlength=-1)
+
+
+@testing.gpu
+@testing.parameterize(*testing.product(
+    {'bins': [
+        # Test monotonically increasing with in-bounds values
+        [0, 1, 2, 4, 10],
+        [1, 2, 3, 4, 5, 6, 7, 8, 9, 10],
+        [0.0, 1.0, 2.5, 4.0, 10.0],
+        [1.5, 2.5, 4.0, 6.0],
```

Comment:
These parameters look the same to me.

Reply:
Is it ok just to test 1 or 2 of them?

Reply:
Please remove all but one.
It doesn't make sense to test parameters with duplicate intention.